phoenix-core/src/it/java/org/apache/phoenix/end2end/CDCBaseIT.java

```diff
@@ -428,8 +434,8 @@ protected List<ChangeRow> generateChanges(long startTS, String[] tenantids, Stri
                         }})
                 ));
                 committer.commit(conn);
-
-                changes.add(addChange(conn, tableName, new ChangeRow(tid, startTS += 100,
+                injectEdge.setValue(startTS += 100);
```

Comment:
This is already getting called by addChange, any specific reason to do this from here? This will get overwritten by next addChange call anyway.

Reply:
I missed that it was called in addChange. I will remove them